custom_components/tuya_local/devices/switch_module_rdcbc_t_j_z_j.yaml

```diff
@@ -0,0 +1,150 @@
+name: Energy monitoring RDCBC
```

Comment:
This config also seems like it should match one of the generic smartplug configs already.